csharp/test/Apache.Arrow.Tests/ArrowFileReaderTests.cs

```diff
@@ -47,6 +48,55 @@ public void Ctor_LeaveOpenTrue_StreamValidOnDispose()
             Assert.Equal(0, stream.Position);
         }
 
+        [Fact]
+        public async Task Ctor_MemoryPool_AllocatesFromPool()
+        {
+            await Ctor_MemoryPool_AllocatesFromPoolHelper(
+                shouldLeaveOpen: false,
+                (stream, memoryPool) => new ArrowFileReader(stream, memoryPool));
+        }
+
+        [Fact]
```

Comment:
nit: you could use theory here and add two test cases: true & false

Reply:
Good call. Fixed.